### PR TITLE
[Fix] Edit recipes now working properly

### PR DESCRIPTION
**Pull request by Lohenrex**
## Description
A user should be able to edit their own recipes. For this purpose, they should be able to see the actual elements of the recipe and decide whether to delete or leave them. In this PR we are preloading the current appliances, ingredients and steps (which do not add themselves as-is).

## Screenshots
A sample recipe edit view with preloaded ingredients and appliances
A sample recipe edit view with preloaded ingredients and appliances

### CODE REVIEW

File: app/javascript/custom/recipes.js [hunk 1]
```diff
@@ -1,85 +1,160 @@
...
+  const ingredientSelect = document.querySelector("#recipe_ingredient_select");
+  const ingredientAmount = document.querySelector("#recipe_ingredient_amount");
+  const ingredientMeasure = document.querySelector("#recipe_ingredient_measure");
+  const applianceSelector = document.querySelector("#recipe_appliance_selector");
+  const isEditPath = document.querySelector("#current_recipe_path");
+  const theForm = document.querySelector("#recipesForm");
+  let selectedIngredients = [];
+  let selectedAppliances = [];
+
+  const deleteAppliance = (e) => { console.log(e)}
```
Comment (itsmeurbi) [inline app/javascript/custom/recipes.js:17]: 👀

**Review by itsmeurbi — CHANGES_REQUESTED** [review]
I got the feeling you don't need to validate every single html element exists on the dom hehe
Let's just get rid of the no needed `console.log` for now

**Review by luisAzcuaga — APPROVED** [review]
LGTM! Nothing else besides Urbi's comment :P

File: app/javascript/custom/recipes.js (hunk 1 above)
Comment (Lohenrex) [reply app/javascript/custom/recipes.js]: Thatś correct! It's gone now :3

**Review by itsmeurbi — APPROVED** [review]

**Pull request merged**